src/main/scala/Executor.scala

```diff
@@ -24,5 +11,5 @@ trait Executor {
 }
 
 case class Result(
-  finalValue: Either[List[Double], List[Int]])
+  finalValue: Either[List[Double], List[Path]])
```

Comment:
PSO always returns a list of double, the list of int was us anticipating ACO I think